### PR TITLE
FEATURE: MPNT-28 Prepare gdc-public-demos repository

### PR DESCRIPTION
**Pull request by lamho-gooddata**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gdc-public-demos-oss-check/2/ : SUCCESS in 8s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gdc-public-demos-oss-check/4/ : SUCCESS in 11s

File: gooddemo/dataSource/dataSource.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "gdc_ds_gooddemo",
+  "name": "GDC Good Demo Data source",
```
Comment (danh-ung) [inline gooddemo/dataSource/dataSource.json:3]: GoodDemo data source

File: gooddemo/databases/ddl/snowflake.ddl [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    "campaign_id"   integer,
+    "customer_id"   integer,
+    "product_id"    integer,
+    "price"         decimal(15, 2),
+    "quantity"      decimal(15, 2),
+    constraint pk_order_lines primary key ("order_line_id"),
+    constraint fk_order_lines_campaign foreign key ("campaign_id") references "campaigns" ("campaign_id"),
+    constraint fk_customer foreign key ("customer_id") references "customers" ("customer_id"),
+    constraint fk_product foreign key ("product_id") references "products" ("product_id")
+);
```
Comment (danh-ung) [inline gooddemo/databases/ddl/snowflake.ddl:52]: newline add the end of file

File: gooddemo/meta.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{ "name": "GoodDemo",
+  "description": "Demo for GoodDemo Organization",
+  "schemaVersion": "1.1.0",
```
Comment (danh-ung) [inline gooddemo/meta.json:3]: "schemaVersion": "1.0.0"

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gdc-public-demos-oss-check/9/ : SUCCESS in 8s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gdc-public-demos-oss-check/11/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gdc-public-demos-oss-check/12/ : SUCCESS in 8s

**Comment by yenkins** [comment]
### Sonar scan result
 - No issue found. Congratulations!

More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gdc-public-demos-gate-PR2

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gdc-public-demos-fossa-scanning-zuul-el7/2/ : SUCCESS in 1m 14s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gdc-public-demos-sonarqube-scan-zuul-gate/2/ : SUCCESS in 40s
- https://checklist.intgdc.com/job/compliance/job/gdc-public-demos-merge-compliance-check/3/ : SUCCESS in 21s

**Pull request merged**